fr.sgued.bodev.json

```diff
@@ -0,0 +1,46 @@
+{
+  "app-id": "fr.sgued.bodev",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "base": "com.system76.Cosmic.BaseApp",
+  "base-version": "stable",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "bodev",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:ro"
```

Comment:
you added an exception for rw but have `ro` here. I think this should be rw instead like all the other cosmic apps.

Reply:
See https://github.com/flathub-infra/flatpak-builder-lint/pull/534

Reply:
Right I forgot to update that. Thanks